ci/azure-pipelines.yml

```diff
@@ -143,7 +143,7 @@ stages:
               mkdir {BUILD,BUILDROOT,RPMS,SOURCES,SPECS,SRPMS}
               mv xpmem-*.tar.* ./SOURCES/
               rpm -qa | grep kernel-devel-
-              for kernel_ver in $(rpm -qa 'kernel-devel-*' --qf "%{version}-%{release}.%{arch}\n"); do
+              for kernel_ver in $(rpm -qa kernel-devel | cut -d'-' -f3-); do
```

Comment:
rpm -qa 'kernel-devel-*'

Reply:
actually we cannot, pls see below, but let me try to re-add `--qf ...`
```
vdi$ rpm -qa kernel-devel
kernel-devel-6.6.12-200.fc39.x86_64
kernel-devel-6.10.10-100.fc39.x86_64

vdi$ rpm -qa 'kernel-devel-*'
kernel-devel-matched-6.10.10-100.fc39.x86_64
<pure kernel-devel is missing here>
```

Reply:
ok, without the trailing - it seems to work

Reply:
pushed with `'kernel-devel*` but seems better to match only on exact `kernel-devel`